[Bug] Crash upon clicking on OpenWeatherMap widget

<!--- Provide a general summary of the issue in the Title above -->
<!--- Check if your issue or something similar has been reported before (if yes upvote/comment there) -->

## Description
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
Launcher crashes and is no longer the default launcher. Can probably be fixed by a simple null-check here: https://github.com/NeoApplications/Neo-Launcher/blob/2dcf7d5f43aa0238bf12bdf053f773bd2e14b7a6/Omega/src/com/saggitt/omega/smartspace/OmegaSmartSpaceController.kt#L259

## Expected Behavior
<!--- Tell us what should happen -->
It should not crash.

## Actual Behavior
<!--- Tell us what happens instead -->
Crash.

## Steps to Reproduce
<!--- Provide a link to a live example (screenshot/recording etc..), or a set of steps to reproduce the issue -->

1. Enable OWM widget
2. Click on temperature
3. Crash


## Device and OS
<!--- Include as many relevant details about the environment you experienced the bug in -->
<!--- For example: -->
<!--- * Device: Samsung Galaxy S10 "beyondlte" -->
<!--- * Android version: 9.0.0 -->
<!--- * Launcher version: 2.1.0 -->
<!--- * Rom: OneUI 1.2 -->


* Device: Oppo K3
* Android version: 11
* Launcher version: 0.9.3
* Rom: Stock


## Logcat
<!--- If possible, include a logcat of the issue, otherwise remove this section-->
```
         AndroidRuntime  D  Shutting down VM
                         E  FATAL EXCEPTION: main
                         E  Process: com.saggitt.omega, PID: 28425
                         E  java.lang.NullPointerException: Attempt to invoke virtual method 'android.os.Bundle android.app.ActivityOptions.toBundle()' on a null object reference
                         E      at com.saggitt.omega.smartspace.OmegaSmartSpaceController.openWeather(OmegaSmartSpaceController.kt:259)
                         E      at com.google.systemui.smartspace.SmartSpaceView$$ExternalSyntheticLambda4.onClick(Unknown Source:2)
                         E      at android.view.View.performClick(View.java:7570)
                         E      at android.view.View.performClickInternal(View.java:7525)
                         E      at android.view.View.access$3900(View.java:836)
                         E      at android.view.View$PerformClick.run(View.java:28680)
                         E      at android.os.Handler.handleCallback(Handler.java:938)
                         E      at android.os.Handler.dispatchMessage(Handler.java:99)
                         E      at android.os.Looper.loop(Looper.java:263)
                         E      at android.app.ActivityThread.main(ActivityThread.java:8296)
                         E      at java.lang.reflect.Method.invoke(Native Method)
                         E      at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:612)
                         E      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1006)
                Process  I  Sending signal. PID: 28425 SIG: 9
```

--- a/Omega/src/com/saggitt/omega/smartspace/OmegaSmartSpaceController.kt
+++ b/Omega/src/com/saggitt/omega/smartspace/OmegaSmartSpaceController.kt
@@ -256,7 +256,7 @@ class OmegaSmartSpaceController(val context: Context) {
             else -> {
                 openURLinBrowser(
                     launcher, data.forecastUrl,
-                    launcher.getViewBounds(v), launcher.getActivityLaunchOptions(v).toBundle()
+                    launcher.getViewBounds(v), launcher.getActivityLaunchOptions(v)?.toBundle()
                 )
             }
         }

